qa/tasks/pool_snaps_thrash.py

```diff
@@ -0,0 +1,105 @@
+"""
+Pool Snaps Thrash
+"""
+import contextlib
+import logging
+import gevent
+import time
+import random
+from gevent import sleep
+from gevent.greenlet import Greenlet
+from gevent.event import Event
+
+log = logging.getLogger(__name__)
+
+class PoolSnapsThrasher(Greenlet, Thrasher, object):
```

Comment:
Same comment as above, I don't think you need to inherit from object since Thrasher does (or should).  Inheriting from Greenlet is likely to be less clear than embedding one.

Reply:
Fixed

Reply:
>  Inheriting from Greenlet is likely to be less clear than embedding one.

When making it similar to other implementations, I also think that the inheriting method is straight forward (no magic involved) and a generic solution to this problem.